### DISCUSSION TITLE
Funding and Contributions

### DISCUSSION THREAD

**Discussion by ncallaway** [AccessKit/accesskit#225] [Q&A] [answered]
First of all I want to say thank you for this project. I was looking into how to make my silly hobby game in Rust accessible, and stumbled across this project. I'm so happy it exists, and think this kind of work will always be under appreciated and under funded.

I'd love to help on that front in whatever small ways I can. First, what's the best way to financially support AccessKit? Is it currently through sponsorships of mwcampbell?

Second, what's the best way for a developer to contribute? I'm coming into the accessibility space pretty cold (I'm familiar with WAG and implementing accessibility basics into websites, but haven't done any native application integration in quite some time). Are there any easy issues to get started with, or any grunt work that can be tackled while I become more familiar with the code base (documentation? Mac OS or Linux integration work? Enigine/third-party application integrations?)

**Comment by mwcampbell** [comment, marked as answer]
Thanks for your interest in supporting this project!
Sponsoring me personally through GitHub Sponsors is currently the best way to fund ongoing work on AccessKit. I'm looking into other options, but that's what I've got so far.

Engine/toolkit and application integrations are a good way to contribute to the project. We also need non-Rust language bindings. @DataTriny is making good progress on a C API, and of course, that's technically enough to allow practically any language to use AccessKit in a roundabout way. But more direct bindings are also welcome.

**Reply by ncallaway** [reply]
Is there any engine or toolkit integration work that's ongoing that needs a hand? Or a list of desired integration targets? I saw that bevy already has support, are the other engines that are next on the list?
